zipline/finance/performance/tracker.py

```diff
@@ -78,28 +78,28 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, env):
+    def __init__(self, sim_params, trading_schedule, env):
```

Comment:
I was planning to leave this as env because the perf tracker uses both the asset finder and the treasury curves that are part of the environment.